flocker/dockerplugin/testtools.py

```diff
@@ -0,0 +1,34 @@
+# Copyright ClusterHQ Ltd.  See LICENSE file for details.
+
+"""
+Testing utilities for ``flocker.dockerplugin`.
+"""
+
+from bitmath import TiB, GiB, MiB, KiB
+
+
+def parse_num(expression):
```

Comment:
the format comes from https://github.com/ClusterHQ/unofficial-flocker-tools/blob/master/design.md which is slightly different than `bitmath.parse_string`, but i can look into if would be applicable.